docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -676,12 +676,90 @@ Start the container `id`
+            "LxcConf": {"lxc.utsname":"docker"},
+            "Memory": 0,
+            "MemorySwap": 0,
+            "CpuShares": 512,
+            "CpusetCpus": "0,1",
+            "PortBindings": { "22/tcp": [{ "HostPort": "11022" }] },
+            "PublishAllPorts": false,
+            "Privileged": false,
+            "ReadonlyRootfs": false,
+            "Dns": ["8.8.8.8"],
+            "DnsSearch": [""],
+            "ExtraHosts": null,
+            "VolumesFrom": ["parent", "other:ro"],
+            "CapAdd": ["NET_ADMIN"],
+            "CapDrop": ["MKNOD"],
+            "RestartPolicy": { "Name": "", "MaximumRetryCount": 0 },
+            "NetworkMode": "bridge",
+            "Devices": [],
+            "Ulimits": [{}],
+            "LogConfig": { "Type": "json-file", Config: {} },
+            "SecurityOpt": [""],
+            "CgroupParent": ""
+        }
+
 **Example response**:
 
         HTTP/1.1 204 No Content
 
 Json Parameters:
 
+-   **Binds** – A list of volume bindings for this container.  Each volume
+        binding is a string of the form `container_path` (to create a new
+        volume for the container), `host_path:container_path` (to bind-mount
+        a host path into the container), or `host_path:container_path:ro`
+        (to make the bind-mount read-only inside the container).
+-   **Links** - A list of links for the container.  Each link entry should be of
+      of the form "container_name:alias".
+-   **LxcConf** - LXC specific configurations.  These configurations will only
+      work when using the `lxc` execution driver.
+-   **PortBindings** - A map of exposed container ports and the host port they
+      should map to. It should be specified in the form
+      `{ <port>/<protocol>: [{ "HostPort": "<port>" }] }`
+      Take note that `port` is specified as a string and not an integer value.
+-   **PublishAllPorts** - Allocates a random host port for all of a container's
+      exposed ports. Specified as a boolean value.
+-   **Privileged** - Gives the container full access to the host.  Specified as
+      a boolean value.
+-   **ReadonlyRootfs** - Mount the container's root filesystem as read only.
+      Specified as a boolean value.
+-   **Dns** - A list of dns servers for the container to use.
+-   **DnsSearch** - A list of DNS search domains
+-   **ExtraHosts** - A list of hostnames/IP mappings to be added to the
+    container's `/etc/hosts` file. Specified in the form `["hostname:IP"]`.
+-   **VolumesFrom** - A list of volumes to inherit from another container.
+      Specified in the form `<container name>[:<ro|rw>]`
+-   **CapAdd** - A list of kernel capabilties to add to the container.
+-   **Capdrop** - A list of kernel capabilties to drop from the container.
+-   **RestartPolicy** – The behavior to apply when the container exits.  The
+        value is an object with a `Name` property of either `"always"` to
+        always restart or `"on-failure"` to restart only when the container
```

Comment:
Hmm, it looks like it could literally be anything, but only `always` and `on-failure` do anything.